cmd/core/cmd_ctx.go

```diff
@@ -16,6 +16,10 @@ func (c CommandContext) AdminClient() service.AdminServiceClient {
 	return c.adminClient
 }
 
+func (c *CommandContext) SetAdminClient(adminClient service.AdminServiceClient) {
```

Comment:
can we add a constructor here
`NewCommandContext(....)` and use that everywhere

Reply:
Done.